templates/news/news-doc-en.html

```diff
@@ -15,22 +15,72 @@
 	<dt>Type</dt>
 	<dd>Canada.ca template</dd>
 	<dt>Last review</dt>
-	<dd>2019-01-30</dd>
+	<dd>2024-09-13</dd>
 	<dt>Conforming to</dt>
 	<dd>Content and IA spec 2.1</dd>
 	<dt>Specification</dt>
 	<dd><a href="https://design.canada.ca/mandatory-templates/news-pages.html#landing">News pages - Canada.ca mandatory template</a></dd>
 	<dt>Template version</dt>
-	<dd>Version 2.0</dd>
+	<dd>Version 2.1</dd>
+	<dt>Available instances</dt>
+	<dd><a href="https://www.canada.ca/en/news.html">https://www.canada.ca/en/news.html</a></dd>
+	<dd><a href="https://www.canada.ca/fr/nouvelles.html">https://www.canada.ca/fr/nouvelles.html</a></dd>
 </dl>
 
 <h2>Working examples</h2>
 <ul>
-	<li><a href="news-en.html">Working example</a></li>
-	<li><a href="news-fr.html" lang="fr">Exemple pratique</a></li>
+	<li><a href="news-en.html">News landing page template - Version 2.1</a></li>
+	<li><a href="news-fr.html" lang="fr">Gabarit de la page d'accueil des nouvelles - Version 2.1</a></li>
+</ul>
+
+<h2>List of changes</h2>
+<ul>
+	<li>Version 2.1:
+		<ul>
+			<li>Addition of randomized functionality for the three items in the features section.</li>
+			<li>Make top promotions more accessible.</li>
+			<li>Update the advanced search link.</li>
+		</ul>
+	</li>
+</ul>
+
+<h3>Previous version</h3>
+<dl>
+	<dt>Version 2.0</dt>
+	<dd><a href="deprecated/news_v2-en.html">News landing page template - Version 2.0</a></dd>
+</dl>
+
+<h2>Transition from version 2.0 to 2.1</h2>
```

Comment:
Is it not a major change because we need to replace content?

```suggestion
<h2>Transition from version 2.0 to 3.0</h2>
```